website/docs/reference/datasources/querying-mysql.md

```diff
@@ -1,5 +1,5 @@
 ---
-sidebar_position: 11
+sidebar_position: 12
```

Comment:
sidebar position is not needed as we are manually generating the sidebar. For now, it's okay.